components/bean/src/main/java/org/switchyard/component/bean/ClientProxyBean.java

```diff
@@ -346,6 +362,25 @@ private Throwable handleException(Exchange exchange, Method method) throws Throw
                 throw BeanMessages.MESSAGES.beanComponentInvocationFailureServiceOperation(_serviceName, method.getName()).setFaultExchange(exchange);
             }
         }
+        /**
+         * This method allows to copy the properties from the current context to the new exchange context
+         *
+         * @param newExchange    the new exchange object where the properties will be copied
+         */
+        private void copyProperties(Exchange newExchange){
+        	ContextProxy context=new ContextProxy();
+        	Context contextNew=newExchange.getContext();
+        	
+            for (org.switchyard.Property p: context.getProperties(Scope.EXCHANGE)) {
+            	if(contextNew.getProperty(p.getName(),Scope.EXCHANGE)==null && !p.getName().startsWith("org.switchyard.bus.camel")){
```

Comment:
To avoid copying internal properties. I don't remember exactly, but this was required.